build/ansible/roles/grafana/files/grafana.ini

```diff
@@ -37,6 +37,8 @@ format = console
 enable_gzip = true
 # The full public facing url
 root_url = https://%(domain)s/graph
+# Serve Grafana from subpath specified in `root_url` setting. By default it is set to false for compatibility reasons.
```

Comment:
Out of curiosity: What was the compatibility reason though (is it with older PMM version?) and do we need to mention it here?